history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -686,19 +752,30 @@ in which I write so plainly to you.
 Farewell, my dear friend, and the Lord be with you.
 I am more than I can say,
 
+[.signed-section-closing]
 Your great lover, and Respectful friend,
 
-W+++.+++ PENN.
+[.signed-section-signature]
+W+++.+++ Penn.
 
+[.postscript]
 I refer you to the enclosed for passages.
 We visited Gichtel and Hooftman, and they us:
 they were at one or two of the meetings at Amsterdam.
 Vale in sternum.
```

Comment:
I couldn't find out what this means so I wasn't sure what to do with it.

Reply:
yeah, i think it must be latin, so maybe let's just add italics and leave it.